src/main.py

```diff
@@ -73,7 +73,7 @@
 if debug:
 	log_level = logging.DEBUG
 else:
-	log_level = logging.INFO
+	log_level = logging.getLevelName(config.get('logging'.upper(), 'INFO'))
```

Comment:
`.get` for default value - sweet! We might use that somewhere else also. I'll think about it.

I do believe however that `.upper()` should be in a different place.